spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe AuthorsController, type: :controller do
```

Comment:
We still miss the test for making sure that it calls the `RequestJob.perform_later` if params are valid. Another useful test is to ensure that it doesn't call `RequestJob.perform_later` if params are invalid (just one combination of wrong input will be enough).